cpp/example_code/s3/get_object.cpp

```diff
@@ -73,8 +73,12 @@ int main()
     Aws::SDKOptions options;
     Aws::InitAPI(options);
     {
-        const Aws::String bucket_name = "my-bucket";
+        //TODO: Name of a bucket in your account.
+        const Aws::String bucket_name = "DOC-EXAMPLE-BUCKET";
+        //TODO: The bucket "DOC-EXAMPLE-BUCKET" must have been previously loaded with "my-file.txt".  The easiest way to achieve
+        //this is to configure and run put_object.cpp's executable first.

```

Comment:
I don't understand the comment. Is it trying to say that the user must run put_object.cpp, with the same object_name value, before running this code?